app/config/sculpin_site.yml

```diff
@@ -91,6 +91,9 @@ subspecies:
     symfony:
         common: "Symfony"
         latin: "symphoniae"
+    upinside:
+        common: "Upinside"
+        latin: "intus autem"
```

Comment:
This should be one word. How do you feel about "intusautemi"?

Reply:
@philipsharp Fixed